### PR TITLE
fix(chromatic cicd): fix over-building chromatic

### PR DESCRIPTION
**Pull request by kishore03109**
## Problem
Chromatic builds keep hitting the quota

## Solution
Comment `!run-chromatic` to trigger chromatic builds  
This is hard to test immediately since this event will only trigger a workflow run iff `chromatic.yml` is in on [default branch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment). To be clear, commenting `!run-chromatic` in this PR here will not trigger the workflow. 

However, this is pretty much a copy paste of the code already existing in for our [e2e triggers](https://github.com/isomerpages/isomercms-frontend/blob/develop/.github/workflows/trigger-e2e.yml) that we know works. 

I have also tried this code in my own [forked repo here](https://github.com/kishore03109/isomercms-frontend/pull/1) as a quick sanity check that this [workflow](https://github.com/kishore03109/isomercms-frontend/blob/develop/.github/workflows/chromatic.yml) works.

### CODE REVIEW

**Comment by kishore03109** [comment]
!run chromatic

**Comment by kishore03109** [comment]
!run-chromatic

**Comment by kishore03109** [comment]
!run-chromatic

**Comment by kishore03109** [comment]
!run-chromatic

**Comment by kishore03109** [comment]
!run-chromatic

**Comment by kishore03109** [comment]
!run-chromatic

**Comment by seaerchin** [comment]
!run-chromatic

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -34,6 +43,23 @@ jobs:
...
+      # Determine if the PR comment should trigger the e2e test suite
+      - name: Check if user is part of Isomer core team
+        uses: tspascoal/get-user-teams-membership@v1
+        id: checkUserMember
+        with:
+          username: ${{ github.actor }}
+          team: ${{ env.ISOMER_CORE_TEAM_SLUG }}
+          GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }} # requires read:org
+
+      - name: Check for trigger words (in PR comment)
```
Comment (seaerchin) [inline .github/workflows/chromatic.yml:55]: i think you missed out the step where this triggers the workflow - see [here](https://github.com/Khan/pull-request-comment-trigger)

what we need is probably (in the next step): 
```
      - run: 'echo Found it!'
        if: steps.check.outputs.triggered == 'true'
```

Comment (kishore03109) [reply .github/workflows/chromatic.yml:55]: Hey, isnt this checked in line 42?

**Comment by kishore03109** [comment]
@seaerchin hey, to be clear right, commenting !run-chromatic will not work here ya since its not in the default branch yet!

File: .github/workflows/chromatic.yml (hunk 1 above)
Comment (kishore03109) [reply .github/workflows/chromatic.yml:55]: As discussed offline, 20d05f4d81ae33e983dd72ecfea8d8be3be42ff3. Thanks for the catch!

**Comment by kishore03109** [comment]
@seaerchin 
oh i think thats cuz the `trigger-e2e.yml` [already](https://github.com/isomerpages/isomercms-frontend/pull/1069) existed in the default branch. To quote the github [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment): Note: This event will only trigger a workflow run if the workflow file is on the default branch. :sadge:

**Comment by harishv7** [comment]
> This is hard to test immediately since this event will only trigger a workflow run iff chromatic.yml is in on [default branch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment). To be clear, commenting !run-chromatic in this PR here will not trigger the workflow.

@kishore03109 what is meant by a default branch? Do you mean like develop or master? Every PR will be on a different branch right (which is not the default branch)? How are we running it on PRs?

**Comment by kishore03109** [comment]
@harishv7 our default branch here is `develop`. Notice that the current `chromatic.yml` file in develop is triggered on a push an open or reopened PR. We are changing the behaviour to be on a comment based trigger. As such we now trigger this on a `issue_comment`, but to quote github [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment), "this event will only trigger a workflow run if the workflow file is on the default branch". Therefore, we can benefit from the functionality of this PR only after a merge to the develop branch. 

Ie, after this pr has been merged, commenting `!run-chromatic` on subsequent PRs should run our chromatic workflow as expected

**Review by seaerchin — APPROVED** [review]

**Comment by harishv7** [comment]
> @harishv7 our default branch here is `develop`. Notice that the current `chromatic.yml` file in develop is triggered on a push an open or reopened PR. We are changing the behaviour to be on a comment based trigger. As such we now trigger this on a `issue_comment`, but to quote github [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment), "this event will only trigger a workflow run if the workflow file is on the default branch". Therefore, we can benefit from the functionality of this PR only after a merge to the develop branch.
> 
> Ie, after this pr has been merged, commenting `!run-chromatic` on subsequent PRs should run our chromatic workflow as expected

Got it!

**Review by harishv7 — APPROVED** [review]

**Pull request merged**